.github/workflows/tests.yaml

```diff
@@ -44,11 +45,11 @@ jobs:
         run: |
           cd go/src/github.com/Workiva/go-datastructures
           go test ./... | tee ${{github.workspace}}/go-test.txt 
-          cd ${{github.workspace}}
-          mkdir artifacts
 
       - name: XML output
         run: |
+          cd ${{github.workspace}}
```

Comment:
🌵 🦐 I think each step starts in `github.workspace`, so I think this cd is superfluous